packages/bp/src/admin/ui/src/app/routes/SegmentHandler.tsx

```diff
@@ -0,0 +1,88 @@
+import segmentPlugin from '@analytics/segment'
+import Analytics, { AnalyticsInstance, PageData } from 'analytics'
+import hash from 'hash.js'
+import React from 'react'
+import { connect } from 'react-redux'
+import { UserState } from '~/user/reducer'
+
+const APP_NAME = 'ADMIN_ANALYTICS' // for reference, in case of second account
+const WRITE_KEY = 'zE0mct7hGOZRtCyImjX9vT1NJ2TpfyGF' // taken from Segment UI
+
+let analytics: AnalyticsInstance
+let loggedIn: boolean = false
```

Comment:
just curious what's that boolean for ? loggedIn through segment ?

Reply:
I renamed it to userIdentified to hopefully make it clearer. I'm doing a one time check, just so I don't blast Segment with events (I could without consequence but it's one less source of data messiness). If there's a pattern you'd like me to use, let me know!